modules/nw-about-configuring-master-interface-container.adoc

```diff
@@ -0,0 +1,175 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_module-type: CONCEPT
+
+[id="nw-about-configuring-master-interface-container_{context}"]
+= About configuring the master interface in the container network namespace
+
+In {product-title} 4.14 and greater the ability to allow users to create a MAC-VLAN, IP-VLAN and VLAN subinterface based on a master interface in a container namespace is now generally available.
+
+This feature allows you to create the master interfaces as part of the pod network configuration in a separate network attachment definition. You can then base the VLAN, MACVLAN or IPVLAN on this interface without requiring the knowledge of the network configuration of the node.
+
+To ensure the use of a container namespace master interface specify the  `linkInContainer` and set the value to `true` in the VLAN, MACVLAN or IPVLAN plugin configuration depending on the particular type of additional network.
+
+An example use case for utilizing this feature is to create multiple VLANs based on SR-IOV VFs. To do so, begin by creating an SR-IOV network and then define the network attachments for the VLAN interfaces.
+
+The following worked example shows how to configure the set up illustrated in this diagram.
+
+.Creating VLANs
+image::345_OpenShift_config_additional_network_0823.png[Creating VLANs]
+
+.Prerequisites
+* SR-IOV Network Operator installed
+* Access to the cluster as a user with the `cluster-admin` role.
+* Install the OpenShift CLI (`oc`).
```

Comment:
If you accept my suggestion on line 24, I'd reword this one to be somewhat more parallel:

```suggestion
* The OpenShift CLI (`oc`) is installed.
```

Reply:
Fixed in line with previous sections https://docs.openshift.com/container-platform/latest/networking/hardware_networks/configuring-sriov-device.html